morf-core/src/main/java/org/alfasoftware/morf/upgrade/ViewDeploymentValidator.java

```diff
@@ -64,6 +64,8 @@ interface Factory  {
 
     /**
      * Creates a {@link ViewDeploymentValidator} implementation for the given connection details.
+     * @param connectionResources connection resources for the datasource.
+     * @return ViewDeploymentValidator.
      */
     ViewDeploymentValidator createViewDeploymentValidator(ConnectionResources connectionResources);
```

Comment:
Do we not need `UpgradeSchemas upgradeSchemas` here?

Ideally we would want to keep the three interfaces relatable to each other, they usually do the job together, whcih makes me wonder why this is happy without checking the schemas.